config/locales/en.yml

```diff
@@ -1649,13 +1651,10 @@ en:
         client_secret: Client Secret
         automatically_approve_accounts: 'Always approve accounts that sign up through this SSO Integration'
         published_admin_portal: Display this SSO Integration on the sign-in page of the admin portal.
-      authentication_provider/github:
-        <<: *authentication_provider
-      authentication_provider/auth0:
-        <<: *authentication_provider
+        skip_ssl_certificate_verification: Skip SSL certificate verification
       authentication_provider/keycloak:
         <<: *authentication_provider
-        skip_ssl_certificate_verification: Do not verify SSL certificate
+        site: Realm
```

Comment:
`<<: *authentication_provider` is not needed in fact, because the only way we access this `site: Realm` translation is by directly fetching `Formtastic::I18n.t('authentication_provider/keycloak.site', scope: :labels)`. But I guess it still makes sense.